scripts/infra/setup-aws.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env -S bash -e
+# SPDX-License-Identifier: Apache-2.0
+
+ensure_system_pkgs() {
+    # shellcheck source=/dev/null
+    source /etc/os-release
+    if [ "$ID" == "fedora" ] || [ "$ID" == "rhel" ]; then
```

Comment:
centos stream?

Reply:
would you happen to know the `$ID` for CentOS? no worries if not, i can launch a VM and check

Reply:
Check for `[ "$ID" == "fedora" ] || [ "$ID_LIKE" == "fedora" ]`. Both CS and RHEL are like Fedora.

Reply:
thanks!

Reply:
Correction, you need to check for `[[ "$ID_LIKE" =~ "fedora" ]]`

```
$ podman run --rm registry.access.redhat.com/ubi9/ubi:9.4 head -n6 /etc/os-release
NAME="Red Hat Enterprise Linux"
VERSION="9.4 (Plow)"
ID="rhel"
ID_LIKE="fedora"
VERSION_ID="9.4"
PLATFORM_ID="platform:el9"

$ podman run --rm quay.io/centos/centos:stream9 head -n6 /etc/os-release
NAME="CentOS Stream"
VERSION="9"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="9"
PLATFORM_ID="platform:el9"

```

Reply:
fixed in most recent patchset